<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Updater</class>
 <widget class="QWidget" name="Updater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>449</height>
   </rect>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="3" column="0" colspan="2">
    <widget class="QTreeView" name="packageView">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QScrollArea" name="distroUpgradesSA">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>448</width>
        <height>94</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="UpdateDetails" name="updateDetails" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UpdateDetails</class>
   <extends>QWidget</extends>
   <header>Updater/UpdateDetails.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
